packages/components/src/InputTime/InputTime.tsx

```diff
@@ -27,7 +26,18 @@ export function InputTime({
       ...params,
     };
 
-    return <FormField type="time" {...fieldProps} />;
+    return (
+      <FormField
+        inputRef={ref}
+        type="time"
+        {...fieldProps}
+        onBlur={handleBlur}
+        onKeyUp={e => {
+          fieldProps.onKeyUp?.(e);
+          !isNaN(parseInt(e.key, 10)) && setTypedTime(prev => prev + e.key);
```

Comment:
Total nit so feel free to ignore and this is something I only learned recently myself, but the `Number()` typescript function is a 'better' way to parse numbers. I put better in quotes because it's subjective and parseInt still has a place when you specifically want an integer and not a float. You don't have to provide the radix argument for Number(), it's faster because it straight up doesn't try to parse garbage, it's either a number or it isnt (123abc is NaN instead of 123). 

Not something for this PR, but something I wanted to let you know of! :)

Also as a bit of an actual nit, these two lines are slightly clunky. Maybe an if statement instead of the inline `&&`? But maybe it's fine and the rest of this PR is just so clean that this stands out a tiny bit. Like, a super tiny amount. I have to comment on SOMETHING. :)